### PR TITLE
Fix logging level

### PR DESCRIPTION
**Pull request by dzikowski**

### CODE REVIEW

File: generators/setup-docker/templates/fabric-config/configtx.yaml [hunk 1]
```diff
@@ -61,7 +62,7 @@ Organizations:
               Rule: "OR('<%= org.mspName %>.admin')"
           Endorsement:
               Type: Signature
-              Rule: "OR('<%= org.mspName %>.peer')"<% } -%>
+              Rule: "OR('<%= org.mspName %>.peer')"<% } %>
```
Comment (Hejwo) [inline generators/setup-docker/templates/fabric-config/configtx.yaml:65]: :+1:

File: generators/setup-docker/templates/fabric-docker/commands-generated.sh [hunk 2]
```diff
@@ -1,32 +1,40 @@
 #!/bin/bash
 
 function prepareChaincodeDirs() {
-  <% chaincodes.forEach(function(chaincode) { %>
-    mkdir -p "$CHAINCODES_BASE_DIR/<%= chaincode.directory %>"
-  <% }) -%>
+  <% if (!chaincodes || !chaincodes.length) { -%>
+    echo "No chaincodes"
```
Comment (Hejwo) [inline generators/setup-docker/templates/fabric-docker/commands-generated.sh:5]: :+1:

File: generators/setup-docker/templates/fabric-docker/docker-compose.yaml [hunk 3]
```diff
@@ -119,7 +119,7 @@ services:<% var rootCaFullName=rootOrg.ca.prefix+"."+rootOrg.organization.domain
     environment:
       - CORE_VM_ENDPOINT=unix:///host/var/run/docker.sock
       - CORE_PEER_ID=<%= peerFullName %>
-      - CORE_LOGGING_PEER=${LOGGING_LEVEL}
+      - FABRIC_LOGGING_SPEC=${LOGGING_LEVEL}
```
Comment (Hejwo) [inline generators/setup-docker/templates/fabric-docker/docker-compose.yaml:122]: By the way we could also investigate and add similar flags to `ca`'s. 
We're not yet consistent across all services.

Comment (dzikowski) [reply generators/setup-docker/templates/fabric-docker/docker-compose.yaml:124]: fortunately it works with the same env var (fixed)

**Review by Hejwo — APPROVED** [review]

**Pull request merged**